managers/sfrRecord.py

```diff
@@ -39,7 +39,8 @@ def mergeRecords(self):
 
         matchedWorks.sort(key=lambda x: x[1])
 
-        allIdentifiers = self.work.identifiers
+        allIdentifiers = self.work.identifiers.copy()
```

Comment:
Is my understanding correct here that by creating a shallow copy and deduping and extending allIdenfitiers, we also update self.work.idenfitiers as well?

Reply:
Is there a case where self.work.identifiers is null?

Reply:
With a shallow copy the original list is only updated if the nested list elements are modified so adding a new element wouldn't change it. I will change this line to a deepcopy instead though so there's no chance of `self.work.identifiers` from being updated alongside allIdentifiers.

Reply:
Cool - I think that sounds good!